app/code/Magento/Catalog/ViewModel/Product/Breadcrumbs.php

```diff
@@ -27,18 +28,26 @@ class Breadcrumbs extends DataObject implements ArgumentInterface
      */
     private $scopeConfig;
 
+    /**
+     * @var Json
+     */
+    private $json;
+
     /**
      * @param Data $catalogData
      * @param ScopeConfigInterface $scopeConfig
+     * @param Json $json
      */
     public function __construct(
         Data $catalogData,
-        ScopeConfigInterface $scopeConfig
+        ScopeConfigInterface $scopeConfig,
+        Json $json
     ) {
         parent::__construct();
 
         $this->catalogData = $catalogData;
         $this->scopeConfig = $scopeConfig;
+        $this->json = $json;
```

Comment:
Hi, @diedburn, due to [Magento backward-compatible guide](http://devdocs.magento.com/guides/v2.2/contributor-guide/backward-compatible-development/) we can't add the required dependency to the constructor method.
thanks